resolwe/observers/models.py

```diff
@@ -29,6 +30,7 @@ class Observer(models.Model):
     """
 
     CHANGE_TYPES = [(change.value, change.name) for change in ChangeType]
+    _suppress_notifications_attribute = "__observer_notification_suppressed"
```

Comment:
Define this constant in the protocol file

Reply:
Done.